Admin/Admin.php

```diff
@@ -2030,12 +2052,15 @@ public function getFilterFieldDescriptions()
     /**
      * {@inheritdoc}
      */
-    public function addChild(AdminInterface $child)
+    public function addChild(AdminInterface $child, $field = null)
     {
         $this->children[$child->getCode()] = $child;
 
         $child->setBaseCodeRoute($this->getCode().'|'.$child->getCode());
         $child->setParent($this);
```

Comment:
Plus, as show in this PR's description, it is possible to work around this limitation like I did.